src/patternfly/components/TextInputGroup/examples/TextInputGroup.md

```diff
@@ -83,7 +83,7 @@ import './TextInputGroup.css'
 
 ### Autocomplete last option hint
 ```hbs
-{{#> text-input-group text-input-group--id="basic" text-input-group--value="apples"}}
```

Comment:
I see you removed `text-input-group--id` here. It looks like the only place we reference `text-input-group--id` is when there is a chip group - https://github.com/patternfly/patternfly/blob/e530391ee29e1b9f719b40881b635914a99fb80e/src/patternfly/components/TextInputGroup/templates/text-input-group--chip-group.hbs#L1

So while it's not needed here, we have `text-input-group--id` in other examples that do not have a chip group. I think we should be consistent - I'd be in favor of adding `text-input-group--id` to all of the examples whether they have a chip group or not, just so it's there if we want to use it.